$NetBSD: patch-doc_Makefile.am,v 1.1 2016/04/17 09:45:59 wiz Exp $

Generate html docs in build phase (instead of install phase).
gnome-doc-tool seems to ignore "-o html" so manually generate
the same effect.

Solves PLIST problem due to missing html files.

This is not know to be reported upstream.

--- doc/Makefile.am.orig	2016-03-07 10:21:06.000000000 +0000
+++ doc/Makefile.am
@@ -7,7 +7,7 @@ EXTRA_DIST = docbook2html.sh docbook
 html/GuideIndex.html: docbook/GuideIndex.xml
 	rm -rf html; mkdir html
 	if [ -x "$(GNOME_DOC_TOOL)" ]; then \
-		"$(GNOME_DOC_TOOL)" html -o html $(srcdir)/docbook/GuideIndex.xml ; \
+		cd html && "$(GNOME_DOC_TOOL)" html $(srcdir)/../docbook/GuideIndex.xml ; \
 	else \
 		echo "gnome-doc-tool not found, html is not built" ; \
 	fi
